docs/_sept_milestone_.md

```diff
@@ -0,0 +1,24 @@
+## Sept Milestone
+
+_With the core of the framework maturing and contributions shifting to the
+LoopBack extensions, we think LoopBack has entered a new stage and it's time to
+revisit our approach to planning work, delivering features, fixing bugs, and
+improving documentation. See our
+[latest blog post](https://strongloop.com/strongblog/2020-community-contribution/)
+for more details._
+
+Here are a list of areas that we'd like to focus on and your contribution is
+more than welcome!
+
+- Blogpost: what are our current plans for juggler modernization #5956
+- [Documentation improvement](https://github.com/strongloop/loopback-next/issues?q=is%3Aissue+is%3Aopen+label%3A2020Q3+label%3Adocs)
+  - Help wanted issues:
+    - Add "Debugging application" in LB4 docs #3051
+    - Better docs for Repository.execute #5264
+- [Internal tooling improvements](https://github.com/strongloop/loopback-next/issues/5669)
+- _a selected list of `help wanted` issues?_
+
```

Comment:
It would be great to invite the maintainers from the community to contribute to the milestone plan too.